react-rust-postgres/backend/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "react-rust-postgres"
+version = "0.1.0"
+authors = ["Jérémie Drouet <jeremie.drouet@gmail.com>"]
+edition = "2018"
```

Comment:
sure?

Reply:
yep :) https://doc.rust-lang.org/cargo/reference/manifest.html#the-edition-field